src/components/DateSelectBarMobile.vue

```diff
@@ -235,115 +179,60 @@ export default {
+          //NOTE
           // range: "this-week"
           range: mode,
         });
       }
     },
-    // updateMessage() {
-    //   this.$store.commit("updateMessage", {
-    //     message: "99999999"
-    //   });
-    // },
-    // toggle() {
-    //   alert("警告，前有BUG");
-    //   this.$store.commit("increment");
-    //   console.log(this.$store.state.count); // -> 1
-    //   var elmnt = document.getElementById("EndItmes");
-    //   document.getElementById("Select").style.top =
-    //     elmnt.clientTop + elmnt.height;
-    //   document.getElementById("Select").style.Left = elmnt.clientLeft;
-    // },
+    ABtime(YYYYMMDD) {
+      return (
+        new Date(YYYYMMDD).getUTCFullYear() +
+        "-" +
+        new Date(YYYYMMDD).getUTCMonth() +
+        "-" +
+        new Date(YYYYMMDD).getUTCDate()
+      );
+    },
+    ontime() {
+      return (
+        new Date().getUTCFullYear() +
+        "-" +
+        new Date().getUTCMonth() +
+        "-" +
+        new Date().getUTCDate()
+      );
+    },
     handleChange() {
       console.log(this.dateValue); // -> 1
       this.$store.commit("setSelectDate", this.dateValue);
       let start = this.dateValue[0];
-      // new Date(this.dateValue[0]).getFullYear() +
-      // "-" +
-      // new Date(this.dateValue[0]).getMonth() +
-      // "-" +
-      // new Date(this.dateValue[0]).getDate();
       let end = this.dateValue[1];
-      // new Date(this.dateValue[1]).getFullYear() +
-      // "-" +
-      // new Date(this.dateValue[1]).getMonth() +
-      // "-" +
-      // new Date(this.dateValue[1]).getDate();
       this.$store.commit("updateDateformat", [start, end]);
-      // console.log("$store", this.$store.state.selectDate); // -> 1
-      // console.log("$store", this.$store.state.selectDate); // -> 1
-
-      // this.$store.dispatch("fetchSummaryApi", {
-      //   start_date: new Date(start).toISOString().substring(0, 10),
-      //   end_date: new Date(end).toISOString().substring(0, 10)
-      // });
-     if (start == end && start == new Date().toISOString().substring(0, 10)) {
+      if (this.ABtime(start) == this.ABtime(end)) {
+        console.log("起始日=截止日", this.ABtime(start), this.ABtime(end));
         //FIXME
-        this.$store.dispatch("fetchToDaysApi");
-        console.log(
-          "起始日=截止日=今日",
-          new Date().toISOString().substring(0, 10)
-        );
-        // if (
-        //   // new Date(start).getUTCMonth() == new Date().getMonth() &&
-        //   // new Date(start).getUTCDate() == new Date().getUTCDate()
-        //   new Date(start).getMonth() == new Date().getMonth() &&
-        //   new Date(start).getDate() == new Date().getDate()
-        // ) {
-        //   console.log("$start == end tore", start, end); // -> 1
-        //   //等於當日 FIX ISSUE 50
-        //   this.$store.dispatch("fetchToDaysApi");
-        // }
+        this.$store.dispatch("fetchSummaryToDaysApi", {
+          start_date: String(start),
```

Comment:
No need to use String constructor to convert value to String, `start` and `end` are strings already